profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestBuildResponse.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2010-2013 Ning, Inc.
+ * Copyright 2014 Groupon, Inc
+ * Copyright 2014 The Billing Project, LLC
+ *
```

Comment:
Indeed - no need to add previous years. Just `Copyright 2016 The Billing Project, LLC`.